antipodal_gripper_controller/src/antipodal_gripper_action_controller_parameters.yaml

```
@@ -0,0 +1,61 @@
+antipodal_gripper_action_controller:
+  action_monitor_rate: {
+    type: double,
+    default_value: 20.0,
+    description: "Hz",
+    validation: {
+      gt_eq: [ 0.1 ]
+    },
+  }
+  joint: {
+    type: string,
+    default_value: "",
```

```suggestion
    read_only: true,
```

I would remove the default empty value for the `joint` parameter